drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -69,7 +70,8 @@ function DragAndDropTemplates(configuration) {
             'draggable': !item.drag_disabled,
             'aria-grabbed': item.grabbed,
             'data-value': item.value,
-            'tabindex': item.focusable ? 0 : undefined
+            'tabindex': item.focusable ? 0 : undefined,
+            'aria-live': 'polite'
```

Comment:
@cahrens Yes - it is here to read out changes in SR block with "draggable" / "draggable, dropped" content. It was [explicitly requested by Mark](https://openedx.atlassian.net/browse/TNL-6031?focusedCommentId=230658&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-230658)